src/composite-data/context/performanceChartByAddressContext.tsx

```diff
@@ -0,0 +1,277 @@
+import { LineProps, Serie } from "@nivo/line"
+import {
+  useGetAllTimeTvlByAddressQuery,
+  useGetHourlyTvlByAddressQuery,
+  useGetWeeklyTvlByAdressQuery,
+} from "generated/subgraph"
+import {
+  createContext,
+  Dispatch,
+  FC,
+  SetStateAction,
+  useContext,
+  useEffect,
+  useState,
+} from "react"
+import { OperationContext } from "urql"
+import { getCalulatedTvl } from "utils/bigNumber"
+import {
+  getPrevious24Hours,
+  getPreviousWeek,
+} from "utils/calculateTime"
+import { formatCurrency } from "utils/formatCurrency"
+import {
+  mutateDayDataByAddress,
+  mutateHourlyData,
+  mutateHourlyDataByAddress,
+} from "utils/urql"
+
+export interface DataProps {
+  series?: Serie[]
+  chartProps: Partial<LineProps>
+  asset?: {
+    decimals: number
+    symbol: string
+    __typename: string
+  }
+}
+
+export interface TvlData {
+  yFormatted: string | number
+  xFormatted: string | number
+}
+
+export interface PerformanceChartContext {
+  fetching: boolean
+  data: DataProps
+  setDataHourly: () => void
+  setDataWeekly: () => void
+  setDataAllTime: () => void
+  reexecuteHourly: (
+    opts?: Partial<OperationContext> | undefined
+  ) => void
+  reexecuteWeekly: (
+    opts?: Partial<OperationContext> | undefined
+  ) => void
+  reexecuteAllTime: (
+    opts?: Partial<OperationContext> | undefined
+  ) => void
+  timeArray: {
+    title: string
+    onClick: () => void
+  }[]
+  tvl: TvlData
+  setTvl: Dispatch<SetStateAction<TvlData>>
+}
+
+const hourlyChartProps: Partial<LineProps> = {
+  axisBottom: {
+    format: "%m/%d: %H:%M",
+    tickValues: "every 6 hours",
+  },
+  xFormat: "time:%H:%M",
+  xScale: {
+    type: "time",
+    format: "%H:%M",
+    useUTC: false,
+    precision: "hour",
+  },
+}
+const dayChartProps: Partial<LineProps> = {
+  axisBottom: {
+    format: "%d",
+    tickValues: "every day",
+  },
+  xFormat: "time:%b %d, %Y",
+  xScale: {
+    type: "time",
+    format: "%d",
+    useUTC: false,
+    precision: "day",
+  },
+}
+
+const allTimeChartProps: Partial<LineProps> = {
+  axisBottom: {
+    format: "%d",
+    tickValues: "every 2 days",
+  },
+  xFormat: "time:%b %d, %Y",
+  xScale: {
+    type: "time",
+    format: "%d",
+    useUTC: false,
+    precision: "day",
+  },
+}
```

Comment:
We should move all of these styles into a sibling `styles.ts` file 😁

Reply:
this is not a stying, this is a initial data/placeholder data